core/templates/dev/head/css/oppia.css

```diff
@@ -962,6 +962,11 @@ textarea {
     display: none;
   }
 }
+@media (max-width: 1023px) {
+  .oppia-navbar-hide-on-medium-width {
+    display: none !important;
```

Comment:
We generally try not to use !important, since it overrides the usual order and makes things less maintainable. Is it needed? Could you try to find a way not to use it (e.g. by using a more specific CSS selector?